### PR TITLE
add go1.11, remove go1.9

### LINKED ISSUE #188: Not compiling under go1.11
The tomo binary is not compiling under go1.11.

Reason here: https://github.com/ethereum/go-ethereum/pull/17368

### PR DESCRIPTION
**Pull request by ngtuna**
fix #188

### CODE REVIEW

File: crypto/bn256/cloudflare/gfp_decl.go [hunk 1]
```diff
@@ -15,4 +22,4 @@ func gfpAdd(c, a, b *gfP)
 func gfpSub(c, a, b *gfP)
 
 //go:noescape
-func gfpMul(c, a, b *gfP)
+func gfpMul(c, a, b *gfP)
```
Comment (etienne-napoleone) [inline crypto/bn256/cloudflare/gfp_decl.go]: Have you edited the files with vim?
Seems like you removed all the blank lines at the end of most of the files.
I think it's not critical tho

Comment (ngtuna) [reply crypto/bn256/cloudflare/gfp_decl.go]: You're right. I edited the files with IDE but forgot to do [gofmt](https://golang.org/cmd/gofmt/)

**Review by etienne-napoleone — APPROVED** [review]
ok for me now

**Comment by ngtuna** [comment]
thanks

**Pull request merged**